### PR TITLE
fix(getting-started): use date only type instead of date

### PR DESCRIPTION
**Pull request by amottier**
Starting with version 7.5.0 getting started tutorial need to use the new date only time as in this use case we only care about day, month and year information.
Update also includes change around auto-generation of forms that include now by default the form container.

Closes [BS-16535](https://bonitasoft.atlassian.net/browse/BS-16535)

### CODE REVIEW

File: md/getting-started-tutorial.md [hunk 1]
```diff
@@ -39,10 +38,10 @@ Remember to click **_Save_** to save your work frequently.
 
 In Bonita BPM Studio, start the UI Designer by clicking the icon in the coolbar. This opens the UI Designer in your browser, at the home page.
 
-In the UI Designer home page, click on the **_Create_** button. This brings a pop-up.  
-In the pop-up, set the type to _Application page_ and name to _TravelTool_.  
-The name must not contain spaces or special characters.  
-This name is used in Bonita BPM to identify the page. It is not displayed to the application user.
```
Comment (educhastenier) [inline md/getting-started-tutorial.md:45]: Double spaces at end of line means "carriage return".
Did you really mean to remove all these?

Comment (abirembaut) [reply md/getting-started-tutorial.md:45]: FYI, the result: http://doc.rd.lan:50274/7.5?page=getting-started-tutorial

Comment (amottier) [reply md/getting-started-tutorial.md:45]: Nope, that was a mistake. Thanks for the review. I'll fix that.

Comment (amottier) [reply md/getting-started-tutorial.md:45]: Should be ok now. Still it seems that line break are ignore for content within a list (not a new issue).

File: md/getting-started-tutorial.md [hunk 2]
```diff
@@ -341,28 +337,28 @@ Select the _Manager review_ task, and go to the **Details** panel, **Execution**
 
 1. Create a new operation by clicking **_Add_**.
 2. In the first field, click the down-arrow to display a list of variables on which an operation can be performed, and double-click _travelRequest_.
-3. Click **_Takes value of_**. This opens a popup listing the available operations. Choose **_Use a Java method_**. The popup displays a list of available methods. Click **_setStatus(String) - void_**, and then click **_OK_**.
+3. Click **_Takes value of_** link. This opens a popup listing the available operations. Choose **_Use a Java method_**. The popup displays a list of available methods. Click **_setStatus(String) - void_**, and then click **_OK_**.
```
Comment (abirembaut) [inline md/getting-started-tutorial.md]: I'd say "Click **the** **_Takes value of_** link", but that's just me quibbling

File: md/getting-started-tutorial.md [hunk 3]
```diff
@@ -376,23 +372,17 @@ The process is now defined. The next section shows you how to configure and run
...
+1. Click **_Run_** in the coolbar. This opens a browser window, logs you in to Bonita BPM Portal as Walter Bates (user used for authentication in the default process configuration), and displays the travel request form.
+1. Fill out the form. Make sure that you use the right format for each field in the temporary forms ("yyyy-mm-dd" format for dates and either "true" or "false" value for booleans).
+1. When you have filled out the form, click **_Start_**. This submits the form and starts the process instance.
+1. At the top-right of the Portal window, click the arrow beside Walter's name and choose **_Logout_**.
+1. Log in as Walter's manager, Helen Kelly, with username `helen.kelly` and password `bpm`. The **Tasks** view is displayed, where there is a task called _Manager review_.
+1. Select the _Manager review_ task. The temporary form for the manager review is displayed in the right hand panel.  
+It contains fields for the items defined in the contract for this step, but not the information that Walter entered in the request.  
 When you create your own forms, you will add this information, so that the manger can see the details of the request before approving or refusing it.
```
Comment (abirembaut) [inline md/getting-started-tutorial.md]: so that the **manager**

File: md/getting-started-tutorial.md [hunk 4]
```diff
@@ -464,19 +446,17 @@ Now define the form for the _Manager review_ task. Start by automatically genera
...
-Set the **Level** property to Level 4, so it is the same size as the title of the form for creating a request.
+7. All of the other widgets you will add will be placed inside the form container (identified with a dashed border).
+8. Add a title widget and set its text property to `Review travel request`. Set the **Level** property to Level 4, so it is the same size as the title of the form for creating a request.
 9. Add read-only widgets to display the details of the travel request, using the following information:
 
    | Label  | Widget type  | Value binding  | Read-only  |
    | -----  | ------------ | -------------- | ---------- |
    | Destination  | Input  | request.destination  | yes  |
-   | Departure date  | Input  | request.departureDate \| date  |yes  |
+   | Departure date  | Date  | request.departureDate | date  |yes  |
```
Comment (abirembaut) [inline md/getting-started-tutorial.md]: Technically, the widget name is Date picker

File: md/getting-started-tutorial.md [hunk 5]
```diff
@@ -486,34 +466,39 @@ Set the **Level** property to Level 4, so it is the same size as the title of th
 
     | Property  | Value  |
     | --------- | ------ |
-    | Hidden  | `formInput.status == 'approved'` (click on "bind" icon) |
+    | Hidden  | `formInput.status === 'approved'` (first click on "fx"/"bind" icon) |
```
Comment (abirembaut) [inline md/getting-started-tutorial.md]: I would say : 
click on the "fx" binding icon first

File: md/getting-started-tutorial.md [hunk 6]
```diff
@@ -486,34 +466,39 @@ Set the **Level** property to Level 4, so it is the same size as the title of th
...
+    The dynamic _Hidden_ property will hide the widget when the request status is set to 'approved'.  
     Otherwise, the field will be marked as required and the form validation will prevent the user from refusing a request without a reason.
 12. Remove the Status widget that was added automatically.
 13. Add a Radio buttons widget with the following properties:
 
     | Property  | Value  |
     | --------- | ------ |
     | Label  | Review decision  |
-    | Available values (click on the binding icon next to the value) | statusValues  |
+    | Available values (click on the binding "fx" icon next to the value first) | statusValues  |
```
Comment (abirembaut) [inline md/getting-started-tutorial.md]: click on the "fx" binding icon [next to the input field] first

File: md/getting-started-tutorial.md [hunk 7]
```diff
@@ -486,34 +466,39 @@ Set the **Level** property to Level 4, so it is the same size as the title of th
...
 ## Run process
 
 You can now run the process using the forms that you created. Make sure that you have saved everything in the UI Designer. Then click **_Run_** in the Studio coolbar.
 
-Fill out the request form as Helen, then log in as William and do the review task.    
-When the review form is displayed, it contains the information that Helen entered in the request form. You can approve or refuse the request.
+Fill out the request form as Walter, then log in as Helen and do the review task.  
+When the review form is displayed, it contains the information that Walter entered in the request form. You can approve or refuse the request.
+
+Once process is finished you can see the request information as stored in the test database in Bonita Studio:
```
Comment (abirembaut) [inline md/getting-started-tutorial.md]: Is "test database" a term we use in the rest of the documentation ?
I would rather say "development database", but I am not sure

File: md/getting-started-tutorial.md [hunk 8]
```diff
@@ -486,34 +466,39 @@ Set the **Level** property to Level 4, so it is the same size as the title of th
...
+1. From the menu bar, choose **Development** menu, choose **Business Data Model**, and then choose **Brows data (h2 console)...**. This opens the h2 (test database) web interface in a web browser.
```
Comment (abirembaut) [inline md/getting-started-tutorial.md]: (development database) administration interface ?

**Review by abirembaut — COMMENTED** [review]
Just a few comments. Other than that it looks good to me. Good job !

File: md/getting-started-tutorial.md [hunk 9]
```diff
@@ -486,34 +466,39 @@ Set the **Level** property to Level 4, so it is the same size as the title of th
...
+    The dynamic _Hidden_ property will hide the widget when the request status is set to 'approved'.  
     Otherwise, the field will be marked as required and the form validation will prevent the user from refusing a request without a reason.
 12. Remove the Status widget that was added automatically.
 13. Add a Radio buttons widget with the following properties:
 
     | Property  | Value  |
     | --------- | ------ |
     | Label  | Review decision  |
-    | Available values (click on the binding icon next to the value) | statusValues  |
+    | Available values (click on the "fx" binding icon next to the to the input field first) | statusValues  |
```
Comment (abirembaut) [inline md/getting-started-tutorial.md]: "to the" is repeated twice

**Review by abirembaut — CHANGES_REQUESTED** [review]
LGTM
Just one typo

**Review by abirembaut — APPROVED** [review]

**Pull request merged**